### ISSUE TITLE
Feature request: provide a way to disable warnings

### ISSUE THREAD

**Issue by BernieSumption** [asvetliakov/typescript-snapshots-plugin#9]
Some projects define tests in a way that trips up the static analysis used to tie test cases to snapshots, causing active snapshots to be incorrectly marked as redundant. My project uses a wrapper around `it()`. #1 is another example of such a configuration.

A simple work-around that would help me a lot is the ability to turn off the "The snapshot is redundant" warnings. That way, at least you don't get false positive warnings.

**Comment by asvetliakov** [comment]
Hello, looks like you're using [snapshot-tools](https://github.com/asvetliakov/snapshot-tools) VS code extension.
If your project uses Typescript you can get rid off the vscode extension and use the TS language plugin instead (this repo). It doesn't have any validation.
I also have plan to replace snapshot-tools extension with bundled TS plugin. Give me a few days.

**Comment by asvetliakov** [comment]
Warnings are disabled starting from ```0.1.0``` version of vscode plugin.
